src/commands/synthetics/run-test.ts

```diff
@@ -218,3 +230,61 @@ export const getTestsList = async (
 
   return testsToTrigger
 }
+
+export const execute = async (
+  runConfig: WrapperConfig,
+  {
+    jUnitReport,
+    reporters,
+    runId,
+    suites,
+  }: {
+    jUnitReport?: string
+    reporters?: (SupportedReporter | Reporter)[]
+    runId?: string
+    suites?: Suite[]
+  }
+) => {
```

Comment:
nit: could you add an explicit return type here? (we usually have a lint rule for that but this repo doesn't have one)